temporalnexus/operation.go

```diff
@@ -22,11 +22,32 @@ import (
 
 	"github.com/nexus-rpc/sdk-go/nexus"
 	"go.temporal.io/api/common/v1"
+
 	"go.temporal.io/sdk/client"
 	"go.temporal.io/sdk/internal"
+	"go.temporal.io/sdk/internal/common/metrics"
+	"go.temporal.io/sdk/log"
 	"go.temporal.io/sdk/workflow"
 )
 
+// GetMetricsHandler returns a metrics handler to be used in a Nexus operation's context.
+func GetMetricsHandler(ctx context.Context) metrics.Handler {
+	nctx, ok := internal.NexusOperationContextFromGoContext(ctx)
+	if !ok {
+		panic("temporalnexus GetMetricsHandler: Not a valid Nexus context")
+	}
+	return nctx.MetricsHandler
+}
+
+// GetLogger returns a logger to be used in a Nexus operation's context.
+func GetLogger(ctx context.Context) log.Logger {
```

Comment:
@bergundy I forgot, Does go SDK nexus have interceptors yet? The activity equivalents for these calls are interceptable. If it doesn't  support interceptors yet then no worries.

Reply:
I didn't see any interceptor equivalent for Nexus yet

Reply:
We have interceptors from the workflow not for the handler yet.

https://github.com/temporalio/sdk-go/issues/1474